KenticoCloud.Delivery/KenticoCloud.Delivery.csproj

```diff
@@ -28,9 +28,11 @@
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
   </PropertyGroup>
-
+	
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.9.9" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.2" />
```

Comment:
As this is a library, please use as low version numbers as possible (1.0.0) - not to force developers to upgrade their other NuGet dependencies if not necessary.